support-level-label.php

```diff
@@ -0,0 +1,408 @@
+<?php
+
+/*
+ * Fetch meta-data for repository from
+ * repo-meta API, cache the results in memory.
+ */
+function vipgoci_repo_meta_api_data_fetch(
+	$repo_meta_api_base_url,
+	$repo_meta_api_user_id,
+	$repo_meta_api_access_token,
+	$repo_owner,
+	$repo_name
+) {
+	$cached_id = array(
+		__FUNCTION__, $repo_meta_api_base_url, $repo_meta_api_user_id,
+		$repo_meta_api_access_token, $repo_owner, $repo_name
+	);
+
+	$cached_data = vipgoci_cache( $cached_id );
+
+	vipgoci_log(
+		'Fetching repository meta-data from repo-meta API' .
+			vipgoci_cached_indication_str( $cached_data ),
+
+		array(
+			'repo_meta_data_base_url'	=> $repo_meta_api_base_url,
+			'repo_meta_user_id'		=> $repo_meta_api_user_id,
```

Comment:
Why are keys here named differently to the variable names that are supplying the values?

Reply:
Thanks for pointing this out! This is leftovers from previous versions, forgot to rename.

I have updated this in 6c3923a.